Cargo.toml

```diff
@@ -10,5 +10,5 @@ crate-type = ["cdylib"]
 [dependencies]
 serde_json = "1.0"
 libc = "0.2"
-jsonpath = "0.1.1"
+jsonpath_lib = "0.1.11"
```

Comment:
I took a look at the Git repo for this crate; I assume you moved to it because it has additional features? Or is it better in some other way?

Reply:
the other project seems a bit dead and this one is working on extending the support for updates